qt/widgets/instrumentview/src/InstrumentWidgetRenderTab.cpp

```diff
@@ -141,6 +141,11 @@ InstrumentWidgetRenderTab::InstrumentWidgetRenderTab(
   m_displayAxes->setCheckable(true);
   m_displayAxes->setChecked(true);
   connect(m_displayAxes, SIGNAL(toggled(bool)), this, SLOT(showAxes(bool)));
+  m_volumeRender = new QAction("VolumeRender", this);
```

Comment:
Has this accidentally been included. I think this is another issue being dealt with here?